client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -39,6 +40,16 @@ const RobotOverview = () => {
    * @description Equivalent to ComponentDidMount in class based components
    */
   useEffect(() => {
+    socket.emit('joinUserRoom', userId);
+    socket.on('successUserRoomConnection', (message) => {
+      console.log(message);
+    });
+    socket.on('errorUserRoomConnection', (message) => {
+      console.log(message);
+    });
+    socket.on('newClientJoinedUserRoom', (message) => {
+      console.log(message);
+    });
```

Comment:
Are those console.log statements really necessary? Otherwise please remove them.

Reply:
+1

Reply:
No they are not. I just already wanted to create the event handlers as they will be important in the following user stories. If you are okay with it I will keep the functions but delete the console.log statements